prompts/cot_agent_prompts/act.txt

```diff
@@ -0,0 +1,25 @@
+You have this information about an agent called <input1>:
+
+<input1>'s world understanding: <input2>
+
+Observed changes in the game state:
+<input14>
+
+You are currently viewing a portion of the map, and from your position at <input6> you observe the following:
+<input5>
+
+Define what should be the nex action for Laura get closer to achieve its goals following the current plan.
```

Comment:
Acá se propagó el error de "nex action for Laura".
Debería ser next action for <input1>

Reply:
Listo! hice esa y otra pequeñas correcciones al prompt.